mftf/2.3/test/actions.md

```diff
@@ -2027,9 +2032,9 @@ See [seeNumberOfElements docs on codeception.com](http://codeception.com/docs/mo
 
 Attribute|Type|Use|Description
 ---|---|---|---
-`selector`|string|optional|
-`userInput`|string|optional|
-`parameterArray`|string|optional|
+`selector`|string|optional| The selector identifying the corresponding HTML element.
+`userInput`|string|optional| Number of instances of the specified selector to be found.
+`parameterArray`|string|optional| Array of parameters to be searched for within the selector.
```

Comment:
select within the range of the result elements. e.g. [0, 10], select between 0 and 10 elements